src/spikeinterface/extractors/neoextractors/intan.py

```diff
@@ -53,6 +53,12 @@ def __init__(
         )
 
         self._kwargs.update(dict(file_path=str(Path(file_path).absolute())))
+        import packaging
+        import neo
+
+        neo_version = packaging.version.parse(neo.__version__)
+        if neo_version > packaging.version.parse("0.13.1"):
+            self._kwargs.update(dict(ignore_integrity_checks=ignore_integrity_checks))
```

Comment:
This should be done in line 45 also no ?

Reply:
It is already handled inside the mapping function:

https://github.com/SpikeInterface/spikeinterface/blob/5ae3a24a467c821d7eb7a5be61944874d27740ca/src/spikeinterface/extractors/neoextractors/intan.py#L70-L75